src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -21,29 +21,24 @@
  */
 final class ComponentAttributes
 {
+    private bool $normalized = false;
```

Comment:
This normalization tracking is required because of how #1041 was implemented. (it first sets all passed values as attributes, then removes the ones that are defined)